chapter-09/recipe-03/cxx-example/test.py

```diff
@@ -1,3 +1,7 @@
+import os
+import sys
+sys.path.append(os.getenv('ACCOUNT_MODULE_PATH'))
+
 from account import pyAccount as Account
```

Comment:
Can you add `# isort: skip` at the end of this line? So that my Emacs configuration doesn't mess with the custom `sys.path.append`